src/wazuh_db/wdb_parser.c

```diff
@@ -336,6 +336,52 @@ int wdb_parse(char * input, char * output) {
             return -1;
         }
         return result;
+    } else if(strcmp(actor, "mitre") == 0) {
+        query = next;
+
+        if(wdb = wdb_open_mitre(), !wdb) {
+            merror("Couldn't open DB mitre");
+            snprintf(output, OS_MAXSTR + 1, "err Couldn't open DB mitre");
+            return -1;
+        }
+
+        if (next = wstr_chr(query, ' '), !next) {
+            mdebug1("Invalid DB query syntax.");
+            mdebug2("DB query error near: %s", query);
+            snprintf(output, OS_MAXSTR + 1, "err Invalid DB query syntax, near '%.32s'", query);
+            return -1;
+        }
+        *next++ = '\0';
+
+        if (strcmp(query, "sql") == 0) {
+            if (!next) {
+                mdebug1("Mitre DB Invalid DB query syntax.");
+                mdebug2("Mitre DB query error near: %s", query);
+                snprintf(output, OS_MAXSTR + 1, "err Invalid DB query syntax, near '%.32s'", query);
+                result = -1;
+            } else {
+                sql = next;
+
+                if (data = wdb_exec(wdb->db, sql), data) {
+                    out = cJSON_PrintUnformatted(data);
+                    snprintf(output, OS_MAXSTR + 1, "ok %s", out);
+                    free(out);
```

Comment:
Hello @danimegar,

Please, use `os_free` instead of `free`.

Regards.

Reply:
Changed free for os_free.